Module 1 - Day 2 - Hello Microverse Page

lmunozm1702 opened the pull request:
In this request I:

- Create the index.html page.
- Create the style.css file to set the color of the content in index.html.
- Set up .gitignore file to avoid tracking the test.md file.

lmunozm1702 committed:
add linter

add config files to CSS&HTML linter.

lmunozm1702 committed:
Create README.md

lmunozm1702 committed:
Create homepage

Add index.html file with the content required.

lmunozm1702 committed:
Create stylesheet

add /assets/css/styles.css

modify index.html:
- Correct the link to stylesheet (the path need to include css folder: /assets/css)
- Set the class h1-red to h1 tag.

lmunozm1702 committed:
Create test.md

Add test.md file into the project folder.

Modify .gitignore file to include the test.md file.

aimalamiri commented on .gitignore:
```diff
@@ -0,0 +1 @@
+test.md
```
- [x] Good job! According to the project requirements, it's important to add the `node_moduels` directory to your `.gitignore` file, so make sure to include it.

lmunozm1702 replied:
It's done.

aimalamiri commented on README.md:
```diff
@@ -0,0 +1,14 @@
+##Getting Started
+This project's about how to create a project on GitHub and set up a linter for HTML&CSS, and make a new branch to implement the "hello microverse" functionality.
+
+##Authors
+###Luis Muñoz
+
+- GitHub: @lmunozm1702
```
- [x] Please add links to your GitHub profile so that people can visit your profile by clicking on the link.

lmunozm1702 replied:
It's done.

aimalamiri reviewed (changes requested):
Hi @lmunozm1702, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![AlmostThereReachGIF](https://user-images.githubusercontent.com/34779928/191347504-9f70659c-a01f-48a1-ad46-b8722d46ffca.gif)

## Highlights


## Required Changes ♻️

- [x] **Please make sure to add the `.hintrc` and the `.stylelintrc.json` files to your repository.**


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 committed:
Code Review: Add linter files

Add:
.hintrc
.stylelintrc.json

As requested on first item on the code review.

lmunozm1702 committed:
Code Review: Update .gitignore

Include node_modules folder into the .gitignore file.

lmunozm1702 committed:
Code Review: Update README.md

Add a link to author's GitHub profile.

Add a space after de # tag to permit the format of the headings.

lmunozm1702 commented:
All code review issues (3) are solved.

enniopo commented:
Hi @lmunozm1702, please request another code review request from the Dashboard. You should be able to do it now. Thanks 🙏

lmunozm1702 commented:
Hi @enniopo, I did the new request from the dashboard, thanks a lot!

mckent05 reviewed (approved):
Hi @lmunozm1702, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and good job! 🎉 

## Optional suggestions
**[OPTIONAL]** - Please note that in your future projects, having a descriptive PR title will help to make your repo look more professional. in this case, for example, your title could be something as short as "Hello Microverse Project" 😄 😄 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 commented:
Hi! @mckent05, it's merged!!! Thanks a lot!!!
